### PR TITLE
[Communication]: Updated SMS Samples and README

### PR DESCRIPTION
**Pull request by lsundaralingam**

### CODE REVIEW

File: sdk/communication/azure-communication-sms/samples/send_sms_to_single_recipient_sample_async.py [hunk 1]
```diff
@@ -7,65 +7,52 @@
...
 """
-FILE: sms_sample_async.py
+FILE: send_sms_to_single_recipient_sample_async.py
 DESCRIPTION:
-    These samples demonstrate sending mutiple sms messages and resending 
-    any failed messages.
-    
-    ///authenticating a client via a connection string
+    This sample demonstrates sending an SMS message to a single recipient. The SMS client is 
+    authenticated a client using a connection string.
```
Comment (beltr0n) [inline sdk/communication/azure-communication-sms/samples/send_sms_to_single_recipient_sample_async.py:13]: see above

File: sdk/communication/azure-communication-sms/samples/send_sms_to_multiple_recipients_sample.py [hunk 2]
```diff
@@ -7,27 +7,28 @@
...
 """
-FILE: sms_sample.py
+FILE: send_sms_to_multiple_recipients_sample.py
 DESCRIPTION:
-    These samples demonstrate sending mutiple sms messages and resending 
-    any failed messages.
-    
-    ///authenticating a client via a connection string
+    This sample demonstrates sending an SMS message to multiple recipients. The SMS client is 
+    authenticated a client using a connection string.
```
Comment (beltr0n) [inline sdk/communication/azure-communication-sms/samples/send_sms_to_multiple_recipients_sample.py:13]: I think this is ungrammatical, looks like we have redundant "a client".

File: sdk/communication/azure-communication-sms/samples/send_sms_to_multiple_recipients_sample_async.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: send_sms_to_multiple_recipients_sample_async.py
+DESCRIPTION:
+    This sample demonstrates sending an SMS message to multiple recipients. The SMS client is 
+    authenticated a client using a connection string.
```
Comment (beltr0n) [inline sdk/communication/azure-communication-sms/samples/send_sms_to_multiple_recipients_sample_async.py:13]: see above

File: sdk/communication/azure-communication-sms/samples/send_sms_to_single_recipient_sample.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: send_sms_to_single_recipient_sample.py
+DESCRIPTION:
+    This sample demonstrates sending an SMS message to a single recipient. The SMS client is 
+    authenticated a client using a connection string.
```
Comment (beltr0n) [inline sdk/communication/azure-communication-sms/samples/send_sms_to_single_recipient_sample.py:13]: see above

**Review by beltr0n — CHANGES_REQUESTED** [review]
grammar fix for comments

**Review by jbeauregardb — APPROVED** [review]
Addressing Bert's comments should be enough 👍🏻

File: sdk/communication/azure-communication-sms/samples/send_sms_to_multiple_recipients_sample.py (hunk 2 above)
Comment (lsundaralingam) [reply sdk/communication/azure-communication-sms/samples/send_sms_to_multiple_recipients_sample.py:13]: Yeah, it should be ```The SMS client is authenticated using a connection string```.

**Review by beltr0n — APPROVED** [review]

**Pull request merged**